### PR TITLE
add in repositories and browser page

### LINKED ISSUE #10: List repositories and components of addons
The docs don't have much detail on this yet, at the very least a link to all the repositories would be good.

### PR DESCRIPTION
**Pull request by andymckay**
* fixes #10

### CODE REVIEW

File: docs/repositories.rst [hunk 1]
```diff
@@ -0,0 +1,47 @@
+Repositories
+============
+
+mozilla-central
+---------------
+
+Browser code is maintained in the mozilla-central. This includes several areas, related to addons. All of the following bug trackers are in Bugzilla:
```
Comment (mstriemer) [inline docs/repositories.rst]: `related to addons` ~> `related to add-ons`.

Comment (mstriemer) [inline docs/repositories.rst]: Also no comma before that (I think?).

Comment (mstriemer) [inline docs/repositories.rst]: > Browser code is maintained in the mozilla-central.

This shouldn't have the word `the` in it or it should the be the `repository` or somthing.

File: docs/repositories.rst [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+Addons manager
```
Comment (mstriemer) [inline docs/repositories.rst]: Add-ons manager?

File: docs/repositories.rst [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+Add-on SDK
+~~~~~~~~~~
+A library and SDK for building extensions. In bugzilla - product: Add-on SDK. `Existing bugs <https://bugzilla.mozilla.org/buglist.cgi?&bug_status=UNCONFIRMED&bug_status=NEW&bug_status=ASSIGNED&bug_status=REOPENED&product=Add-on%20SDK>`_ or `file a new bug <https://bugzilla.mozilla.org/enter_bug.cgi?product=Add-on%20SDK>`_. Documentation `on MDN <https://developer.mozilla.org/en-US/Add-ons/SDK>`_.
+
+WebExtensions
+~~~~~~~~~~~~~
+An API for building add-ons that works with e10s and is compatible with Google Chrome. In bugzilla - product: Toolkit, component: WebExtensions. `Existing bugs <https://bugzilla.mozilla.org/buglist.cgi?bug_status=UNCONFIRMED&bug_status=NEW&bug_status=ASSIGNED&bug_status=REOPENED&component=WebExtensions&product=Toolkit>`_ or `file a new bug <https://bugzilla.mozilla.org/enter_bug.cgi?product=Toolkit&component=WebExtensions>`_. Documentation `on MDN <https://developer.mozilla.org/en-US/Add-ons/WebExtensions/>`_.
```
Comment (mstriemer) [inline docs/repositories.rst]: No shout out for Opera?

Comment (andymckay) [inline docs/repositories.rst]: :)

File: docs/repositories.rst [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+github
```
Comment (mstriemer) [inline docs/repositories.rst]: I think all these "github"s should be "GitHub"s.

Comment (andymckay) [inline docs/repositories.rst]: sure

**Comment by andymckay** [comment]
Switching _ to __ to avoid "Duplicate explicit target name" in some of these.

**Comment by mstriemer** [comment]
I was trying to figure out how to fix that, too. What does `__` do?

**Comment by mstriemer** [comment]
r+

**Comment by andymckay** [comment]
http://stackoverflow.com/questions/5464627/how-to-have-same-text-in-two-links-with-restructured-text

**Pull request merged**